components/homme/src/share/prim_driver_base.F90

```diff
@@ -1436,30 +1380,44 @@ subroutine applyCAMforcing_tracers(elem,hvcoord,np1,np1_qdp,dt,adjustment)
   real (kind=real_kind)  :: dpnh_dp_i(np,np,nlevp)
 #endif
 
+#ifdef MODEL_THETA_L
+  if (rsplit==0) then
+     adjust_ps=.true.   ! we are always on ref levels
+  else
+     adjust_ps=.true.   ! eventually change this to false
+  endif
+#else
+  adjust_ps=.true.      ! preqx doesn't use 
```

Comment:
if preqx "doesn't use" does that mean this should be `.false.`?

Reply:
forgot to finish that comment - no improved.